react_demo/cpp/index.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>FROM_C_PLUS_PLUS</title>
```

Comment:
I really don't think it's worth it to invest more time cleaning up this PR.
:-)

On Saturday, June 6, 2015, John Babak <notifications@github.com> wrote:

> In react_demo/cpp/index.html
> <https://github.com/KnowSheet/SimpleDashboard/pull/7#discussion_r31874061>
> :
>
> > @@ -0,0 +1,250 @@
> > +<!DOCTYPE html>
> > +<html>
> > +  <head>
> > +    <meta charset="utf-8">
> > +    <meta http-equiv="x-ua-compatible" content="ie=edge">
> > +    <title>FROM_C_PLUS_PLUS</title>
>
> The original comment was: "This placeholder is not the only thing passed
> from C++, so it should be PAGE_TITLE_FROM_C_PLUS_PLUS." (currently just
> FROM_C_PLUS_PLUS). I saw you did DATA_JSON to DATA_FROM_C_PLUS_PLUS change
> which is OK, but I can't see the same change for PAGE_TITLE.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KnowSheet/SimpleDashboard/pull/7/files#r31874061>.
>


-- 
And, as always, have a nice day!
Dima

http://dimakorolev.com/ | http://www.linkedin.com/in/dimakorolev |
dmitry.korolev@gmail.com | +1 (312) 593 2783